code/findoutlier.py

```diff
@@ -0,0 +1,38 @@
+import numpy as np
+from outlierfunction import outlier
+
+for i in range(1,10):
+    for j in range(1,4):
+        # set general path for reaching dvars and fd files
+        # also path for saving files
+        txtpath='ds005/sub00'+`i`+'/BOLD/task001_run00'+`j`+'/QA/'
```

Comment:
Need to set a path pointing to data directory (before 'ds005...'), for example if this file is in code directory, need to add "../data/"